packages/editor-ui/src/stores/posthog.ts

```diff
@@ -157,6 +157,10 @@ export const usePostHog = defineStore('posthog', () => {
 		if (name === ONBOARDING_EXPERIMENT.name && variant === ONBOARDING_EXPERIMENT.variant) {
 			segmentStore.showAppCuesChecklist();
 		}
+
+		if (name === TEMPLATE_EXPERIMENT.name && variant === TEMPLATE_EXPERIMENT.variant) {
+			settingsStore.disableTemplates();
```

Comment:
does this work well here despite the 2 second debounce? does the templates icon flicker in the menu then disappear? 
We only need the debounce [here](https://github.com/n8n-io/n8n/pull/5963/files#diff-71a4eee50c3824c3d17a6a53f7497ebf292c3a10e712df3fdbc1a8f971e72e3cR135) so maybe we can have two functions.. `evaluateExperiments` and `evaluateExperimentsDebounced`
renamed it to `evaluate` since it's doing more than just tracking now

Reply:
Ahhh, I noticed that but was not sure if we always needed the debounce. I was going to ask you about it today.